### ISSUE TITLE
Does redis stack work with redis cluster?

### ISSUE THREAD

**Issue by Tagarito** [redis-stack/redis-stack#358]
Is redis stack supposed to work with redis cluster?

I created a cluster using redis stack (cluster-enabled yes on config) but most operations seem to not work is this expected?

**Comment by Skoucail** [comment]
Interested in this to.
Would love cluster support for redis stack!

**Comment by chayim** [comment]
Hi @Skoucail . Here are a few links that should help!

1. A [docker-compose](https://github.com/redis/redis-py/blob/master/docker-compose.yml) in redis-py that does exactly this, creates a redis-stack cluster.
2. Note, that it uses the files in [this folder](https://github.com/redis/redis-py/tree/master/dockers) to make things easier.

In the above, I create a cluster, in a single docker, from redis-stack. But I think that should be enough to help you out.

Finally. if you're interested in other possible environment types, I have a repository I maintain [here](https://github.com/chayim/redis-environments/blob/master/docker-compose.yml).

**Comment by Skoucail** [comment]
@chayim 
Thanks for your reply.
I found some time to test it out.
I was successful setting up the cluster itself but i'm running into issues.

1) RediSearch
Seems the module included in the stack is not suited for clusters (or the NRedisStack library doesn't handle it well)
I made a custom redis stack docker with the RediSearch oss build but the coordinator module. (As i'm using this for other deployments without stack)
But this seems to be containing an issue in the current version crashing all replica nodes...

2) RedisTimeSeries
Less issues here. But cluster support for some commands seems to be not fully working. MRANGE/MGET. (Also using the NRedisStack for c# library)
I found a issue in the RedisTimeSeries github saying we need to call timeseries.REFRESHCLUSTER.
Its not realy explaining when this needs to be called and if it needs to be called on all nodes or only masters.
But after calling this (on all nodes) just before executing MRANGE/MGET i'm getting the expected result.

I have no experience with the other plugins contained in Redis Stack. 

Unless i'm missing something or doing stuff wrong this makes me think RedisStack is far from cluster ready...
